src/main/java/uk/gov/cshr/domain/Identity.java

```diff
@@ -41,7 +44,7 @@
     public Identity() {
     }
 
-    public Identity(String uid, String email, String password, boolean active, boolean locked, Set<Role> roles, Instant lastLoggedIn, boolean deletionNotificationSent) {
+    public Identity(String uid, String email, String password, boolean active, boolean locked, Set<Role> roles, Instant lastLoggedIn, boolean deletionNotificationSent, Long failedLoginAttempts) {
```

Comment:
if we have a bit time, I would either introduce a IdentityBuilder, or enable Lombok's annotation e.g @data

Reply:
Not a fan of lombok ;)

will provide a Builder with ObjectMother for tests